web_app/db/seed_data.py

```diff
@@ -153,14 +163,38 @@ def create_vaults(session: SessionLocal, users: list[User]) -> None:
         logger.info("No vaults created.")
 
 
+def create_transaction(session: SessionLocal, positions: list[Position]) -> None:
+    """
+    Create and save fake transaction records to the database.
+    Args:
+        session (Session): SQLAlchemy session object.
+        positions (list): List of Position objects to associate with transactions.
+    """
+    transactions = []
+    for position in positions:
+        for _ in range(2):
+            transaction = Transaction(
+                position_id=position.id,
+                status=fake.random_element(
+                    elements=[status.value for status in TransactionStatus]
```

Comment:
You could move this list generator outside `for` loop and use it twice: 
1) `for _ in range(len(transaction_statuses)):`
2) `elements=transaction_statuses`